action.yml

```diff
@@ -0,0 +1,30 @@
+name: pysa
+description: "Run pysa on the repository and check for vulnerabilities"
+
+inputs:
+  build-type:
+    description: Pysa build to use. Possible values are stable and nightly. Default is stable.
+    required: false
+    default: "stable"
+
+runs:
+  using: "composite"
+  steps:
+    - run: if [ ${{inputs.build-type}} = "nighty" ]; then pip install pyre-check-nightly; else pip install pyre-check; fi
+      shell: bash
+    - run: if [ ! -f .pyre_configuration ]; then printf ".\n" | pyre init; fi
+      shell: bash
+    - run: pyre infer
+      shell: bash
+    - run: pyre infer -i --annotate-from-existing-stubs
```

Comment:
It's a good idea to use `pyre infer`, but since we are using `infer` in-place, we might want to include in docs somewhere that the output from SARIF will be different from their code (e.g. line numbers in traces being off, type annotations that the developers didn't add themselves, etc)

Reply:
Yep makes sense :)